Android.mk

```diff
@@ -0,0 +1,26 @@
+LOCAL_PATH:= $(call my-dir)
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := IchnaeaNlp
+LOCAL_MODULE_TAGS := optional
+LOCAL_PACKAGE_NAME := IchnaeaNlp
+
+ichnaea_root  := $(LOCAL_PATH)
+ichnaea_out   := $(OUT_DIR)/target/common/obj/APPS/$(LOCAL_MODULE)_intermediates
+ichnaea_build := $(ichnaea_root)/build
+ichnaea_apk   := build/outputs/apk/IchnaeaNlp-release-unsigned.apk
```

Comment:
Is it possible to change the gradle output name ?
(When building and integrating a ROM, you cannot deduce the name of the folder without looking at specific project Android.mk...)